pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/BrokerRegistryImpl.java

```diff
@@ -151,7 +151,7 @@ public String getBrokerId() {
     @Override
     public CompletableFuture<List<String>> getAvailableBrokersAsync() {
         this.checkState();
-        return brokerLookupDataLockManager.listLocks(LOOKUP_DATA_PATH).thenApply(Lists::newArrayList);
```

Comment:
why removing `thenApply(Lists::newArrayList);`? Can't the caller update the cache?

Reply:
Oh, yes changed back.